dev/benchmarks/macrobenchmarks/lib/src/web/recorder.dart

```diff
@@ -578,11 +578,11 @@ class Timeseries {
 
   /// The number of frames ignored as warm-up frames, used only
   /// when [useCustomWarmUp] is true.
-  int _warmUpFrameCount;
+  int? _warmUpFrameCount;
```

Comment:
Could this be initialized to 0 here and int be kept non-nullable? Or is it better that it's nullable because there are code paths where this variable isn't used?

Reply:
The original code assumed it was null if it wasn't being used, so I decided to keep this as a guard against it being used when it shouldn't be.